helper/ttl/validation.go

```diff
@@ -0,0 +1,83 @@
+package ttl
```

Comment:
I think any TTL helpers should live in `logical/framework/lease.go` with the other TTL helper functions

Reply:
Agreed, and this would make moot my previous comments re: circular imports from a helper, new interface, etc.